packages/shared/src/components/squads/SquadsToShare.tsx

```diff
@@ -55,6 +55,7 @@ export function SquadsToShare({
     [squads, isLoading, onClick, size, squadAvatarSize],
   );
 
+  if (!hasSquadAccess) return null;
```

Comment:
Do we not need this for logged in users who don't have access yet?
Also not sure how this broke it for anonymous users actually.

Reply:
My bad, thought it was removed 😅